docs/bootstrap.sh

```diff
@@ -54,7 +54,7 @@ case "$cmd" in
   "hash")
     echo "$hash"
     ;;
-  "release")
+  "release-docs")
```

Comment:
this is used in root `bootstrap.sh` so if it's to be changed, `docs` needs to be removed from the list of projects in the `release` action

Reply:
yes you're right, we don't want docs to be "released", they're always flush with master (releases are in `versioned_docs`)